tests/Microsoft.Identity.Test.UIAutomation/XamarinDroidMSALTests.cs

```diff
@@ -64,9 +68,66 @@ public void InitializeBeforeTest()
         }
 
         /// <summary>
-        /// Runs through the standard acquire token flow, using the default app configured UiBehavior = Login
+        /// Test runner to run all tests, as test initialization is expensive.
         /// </summary>
         [Test]
+        public void RunAllTests()
+        {
```

Comment:
sweet

Reply:
It might be better to refactor the initialization of each tests so that the app is initialized only once and the instance of the application is resused by subsequent tests. This way, we will have all tests separated, with the first one opening the app and the rest reusing it. Test reports will be nicer, e.g. we can clearly see if ADFSv3 has a problem because all of our ADFSv3 tests will have failed.